### PR TITLE
Update render.hpp

### PR DESCRIPTION
**Pull request by FarikLoL**
utf8

### CODE REVIEW

**Comment by ifBars** [comment]
+rep - haven't tested but seems right

**Comment by IMXNOOBX** [comment]
There have been no changes other than prettified the code a bit. Could you provide a reason of how this would improve the project

**Comment by ifBars** [comment]
> There have been no changes other than prettified the code a bit. Could you provide a reason of how this would improve the project

Seems like [it uses UTF 8](https://github.com/IMXNOOBX/cs2-external-esp/pull/44/files#diff-80557a2a229be93cb7e2c79548b2544e625ce6d329b9fab5244dc6f5472ea9bdL45) text which I think fixes something or displays certain names that normally can't or somethingo

**Comment by IMXNOOBX** [comment]
You are right. I completely missed that part, thank you for clarifying it. i haven tested it yet but if you have, does it make any difference

**Comment by ifBars** [comment]
> You are right. I completely missed that part, thank you for clarifying it. i haven tested it yet but if you have, does it make any difference

I can confirm this does actually make a difference. When player's have weird characters in their names that Windows' character encoding, _([Windows-1252](https://en.wikipedia.org/wiki/Windows-1252) (a form of ANSI character encoding)) <- what the cheat currently uses_, cannot handle it will display weird things like the image below. That is where Unicode character encoding comes in, as this can display a whole lot more characters than ANSI.
image

- The original code uses TextOutA, which is designed for [ANSI character](https://en.wikipedia.org/wiki/ANSI_character_set) encoding.
- The modified code uses TextOutW, which is designed for Unicode character encoding ([UTF-16](https://en.wikipedia.org/wiki/UTF-16) in this case).

TLDR; Change from TextOutA to TextOutW lets weird player names be rendered correctly
